core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -545,6 +545,7 @@ private <K, V> Cache<K, V> internalGetCache(String cacheName) {
          try {
             return (Cache<K, V>) cacheFuture.join();
          } catch (CompletionException e) {
+            log.errorf(e, "Failed to get cache %s", actualName);
```

Comment:
It needs a method in the `Log` class. Only debug and trace levels do not require it.